docs/source/installation/guide.rst

```diff
@@ -145,10 +154,12 @@ Non-GPU environments
 ~~~~~~~~~~~~~~~~~~~~
 
 For users who want to try ChainerMN in a CPU-only environment,
-typically for testing for debugging purpose, ChainerMN can be built
-with the ``--no-nccl`` flag.::
-
-  $ python setup.py install --no-nccl
+typically for testing for debugging purpose, ChainerMN can be used by not installing CuPy.
```

Comment:
Whole sentence would be like this:

Users who want to try ChainerMN in CPU-only environment  may skip installation of CuPy. Non-GPU set up may not be performant as GPU-enabled set up, but would would be useful for testing or debugging training program in non-GPU environment such as laptops or CI jobs.

Reply:
Thank you for your comment. I will fix it.